mmv1/products/pubsub/Topic.yaml

```diff
@@ -149,3 +154,40 @@ properties:
       set, message retention is controlled by settings on individual subscriptions.
       The rotation period has the format of a decimal number, followed by the
       letter `s` (seconds). Cannot be more than 31 days or less than 10 minutes.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'ingestionDataSourceSettings'
+    description: |
+      Settings for ingestion from a data source into this topic.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'awsKinesis'
+        description: |
+          Settings for ingestion from Amazon Kinesis Data Streams.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'streamArn'
+            description: |
+              The Kinesis stream ARN to ingest data from.
+            required: true
+          - !ruby/object:Api::Type::String
+            name: 'consumerArn'
+            description: |
+              The Kinesis consumer ARN to used for ingestion in
+              Enhanced Fan-Out mode. The consumer must be already
+              created and ready to be used.
+            required: true
+          - !ruby/object:Api::Type::String
+            name: 'awsRoleArn'
```

Comment:
I noticed you aren't updating this value in the tests. Is it immutable? If so, we should add `immutable: true` to make sure that if a user does want to change this part of their config, it will correctly create a new resource with the updated value.

Reply:
Fixed the test to update the value. All these ingestion fields are mutable.